WalletWasabi.Fluent/ViewModels/CoinControl/Core/Cells/LabelsCellViewModel.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using WalletWasabi.Fluent.Helpers;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells;
+
+public class LabelsCellViewModel : ViewModelBase
+{
+	public LabelsCellViewModel(ICoin coin)
+	{
+		Labels = coin.Labels;
+
+		if (coin.Labels == CoinPocketHelper.PrivateFundsText)
+		{
+			PrivacyLevel = Privacy.Private;
+		}
+		else if (coin.Labels == CoinPocketHelper.SemiPrivateFundsText)
+		{
+			PrivacyLevel = Privacy.SemiPrivate;
+		}
+		else
+		{
+			PrivacyLevel = Privacy.NonPrivate;
+		}
+	}
+
+	private enum Privacy
+	{
+		Invalid = 0,
+		SemiPrivate,
+		Private,
+		NonPrivate
+	}
+
+	public bool IsPrivate => PrivacyLevel == Privacy.Private;
```

Comment:
Can you just set IsPrivate , IsSemiPrivate nd IsNonPrivate  from constructor, why we need Privacy enum here ?